frontend/src/helpers/assignment-service/api-wrapper.ts

```diff
@@ -14,7 +14,9 @@ const getAssignmentById = async ({
 }: {
   assignmentId: string;
 }) => {
-  const _response = await api.get(`/assignment/${assignmentId}`);
+  const _response = await api
```

Comment:
What will happen if we comment out this code? As we don't need to make the API call for now, I don't think it makes sense to keep this code here?

Reply:
you get unused -vars errors with api and the imports

Reply:
Oh, then I think for now we can remove the import statement first?

Reply:
nah leave it like this, removing the `api` means removing `await` which means the function cannot be `async` which breaks a lot of your other stuff